package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "6.2.0",
+  "version": "6.3.0",
```

Comment:
this should not be here. If you want it based on 6.2, you must rebase interactively and remove unrelated commits from master

Reply:
Yeah i see the other unrelated. I'll rebase and update